### PR TITLE
Capitalized Boussinesq approximation

### PR DESCRIPTION
**Pull request by bobmyhill**
Addresses Issue #1623

### CODE REVIEW

File: doc/manual/manual.tex [hunk 1]
```diff
@@ -2005,13 +2005,13 @@ \subsubsection{Combined formulations}
 (Section~\ref{sec:tala}).
 
 \item
-``boussinesq approximation'': This formulation sets the mass conservation approximation to ``incompressible'', 
+``Boussinesq approximation'': This formulation sets the mass conservation approximation to ``incompressible'',
```
Comment (bangerth) [inline doc/manual/manual.tex]: this occurrence of the word needs to match what is being used in the .prm files. Did you change that as well?

File: include/aspect/parameters.h [hunk 2]
```diff
@@ -123,7 +123,7 @@ namespace aspect
        */
       enum Kind
       {
-        boussinesq_approximation,
+        Boussinesq_approximation,
```
Comment (bangerth) [inline include/aspect/parameters.h]: I think we should keep it lower-case here, just like we have functions such as `solve_stokes()` etc.

File: include/aspect/parameters.h [hunk 3]
```diff
@@ -141,8 +141,8 @@ namespace aspect
           return Formulation::isothermal_compression;
         else if (input == "anelastic liquid approximation")
           return Formulation::anelastic_liquid_approximation;
-        else if (input == "boussinesq approximation")
-          return Formulation::boussinesq_approximation;
+        else if (input == "Boussinesq approximation")
+          return Formulation::Boussinesq_approximation;
```
Comment (bangerth) [inline include/aspect/parameters.h]: and this changes what we accept in .prm files

File: source/simulator/parameters.cc [hunk 4]
```diff
@@ -322,7 +322,7 @@ namespace aspect
     prm.enter_subsection("Formulation");
     {
       prm.declare_entry ("Formulation", "custom",
-                         Patterns::Selection ("isothermal compression|custom|anelastic liquid approximation|boussinesq approximation"),
+                         Patterns::Selection ("isothermal compression|custom|anelastic liquid approximation|Boussinesq approximation"),
```
Comment (bangerth) [inline source/simulator/parameters.cc]: this is an incompatible change -- we should discuss

File: doc/manual/manual.tex (hunk 1 above)
Comment (bobmyhill) [inline doc/manual/manual.tex]: Yes, that should be changed in all files where it occurs (this PR is just a recursive sed find-replace)

**Comment by gassmoeller** [comment]
Yes, for this change we should include the sed command to fix it into the `doc/update_prm_files_...` script

**Pull request merged**